README.md

```diff
@@ -6,73 +6,68 @@
     <strong>Network-wide ad blocking via your own Linux hardware</strong>
 </p>
 
-The Pi-hole[®](https://pi-hole.net/trademark-rules-and-brand-guidelines/) is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content, without installing any client-side software.
+The Pi-hole® is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content, without installing any client-side software.
```

Comment:
Why did you remove this link?

Reply:
That was unintentional, sorry... let me amend my PR.